cql2pgjson/src/test/java/org/z3950/zing/cql/cql2pgjson/IndexPerformanceTest.java

```diff
@@ -101,7 +101,7 @@ public void cqlValue() throws CQL2PgJSONException, IOException {
     runSqlStatement("DROP INDEX IF EXISTS idx_value;");
     runSqlStatement("CREATE INDEX idx_value ON config_data "
         + "((lower(f_unaccent(jsonb->>'value'))) text_pattern_ops);");
-    CQL2PgJSON cql2pgJsonSchema = new CQL2PgJSON("jsonb", Util.getResource("indexPerformanceTest.json"));
+    CQL2PgJSON cql2pgJsonSchema = new CQL2PgJSON("jsonb");
```

Comment:
rename all subsequent uses of cql2pgJsonSchema to cql2pgJson and remove this line:

    CQL2PgJSON cql2pgJsonSchema = new CQL2PgJSON("jsonb");